### PR TITLE
Lily/SoundExpanded: add play sound at and loop begin/end blocks

### PR DESCRIPTION
**Pull request by CST1229**
image

Adds one *very* highly-requested feature and another probably-less-requested but still useful feature. They work pretty much as you'd expect. They were actually pretty straightforward to implement (other than start sound at requiring either function overriding or function duplication, and I went with the latter).

### CODE REVIEW

File: extensions/Lily/SoundExpanded.js [hunk 1]
```diff
@@ -24,13 +24,43 @@
...
+              SOUND: {
+                type: Scratch.ArgumentType.SOUND,
+              },
+            },
+            extensions: ["colours_sounds"],
+          },
+          {
+            opcode: "startLoopingBegin",
+            blockType: Scratch.BlockType.COMMAND,
+            text: "start looping [SOUND] begin [START]s",
```
Comment (LilyMakesThings) [inline extensions/Lily/SoundExpanded.js:37]: change "s" to "secs", and this is perfect

Comment (CubesterYT) [reply extensions/Lily/SoundExpanded.js]: I would say "seconds" for parity

**Comment by LSPECTRONIZTAR** [comment]
when will it be merged?

**Comment by CST1229** [comment]
> when will it be merged?

when GarboMuffin gets to it (he's the only one who can merge PRs, and also has a job)

**Comment by LSPECTRONIZTAR** [comment]
> > when will it be merged?
> 
> when GarboMuffin gets to it (he's the only one who can merge PRs, and also has a job)

Can't someone contact him and let him know?

**Pull request merged**